.circleci/config.yml

```diff
@@ -50,10 +96,6 @@ jobs:
             docker-compose build
             docker-compose pull db
 
-      - run:
-          name: Linting
-          command: docker-compose run web lint
```

Comment:
Since we aren't doing this anymore, I think we can also remove the lint task from `run.sh`.